### ISSUE TITLE
Feature suggestion: Allow changing font size

### ISSUE THREAD

**Issue by ruedigerkupper** [litewrite/litewrite#295]
I like simple tools, and it is okay that litewrite offers no formatting. I am refering to the displayed font size. On my phone, it appears rather large, which is nice for quick note taking, but I tend to lose my overview on longer texts. Please add a simple option for changing the display font size (like 'larger/smaller'-buttons). Thanks !

**Comment by jorinvo** [comment]
Good point! It's difficult to make things look right on all devices. Happy to accept help with this since I won't have time to look into this myself :)
